src/formatters/deployResultFormatter.ts

```diff
@@ -223,6 +227,8 @@ export class DeployResultFormatter extends ResultFormatter {
         this.ux.log('');
         this.ux.styledHeader(chalk.red(`Component Failures [${failures.length}]`));
         this.ux.table(
+          // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+          // @ts-ignore
           failures.map((entry: FileResponseFailure) => ({
```

Comment:
if you remove the `FileResponseFailure` type it shows almost all of the props it tries to access doesn't exist. Are the types from SDR wrong or where does it get corrected?

Reply:
I couldn't get this to work without refactoring a bunch of stuff, will verify on QA.